src/dynamodb/gameStateDBService.ts

```diff
@@ -151,6 +151,29 @@ export const getInteractionCount = async (gameId: string): Promise<number | unde
 /* ----------------------------------------------------------------------------
  * Update
  * ------------------------------------------------------------------------- */
+
+/*
+ * Add new game state or replace existing game state with the same game id into GAME_STATE_TABLE.
+ * Mainly to be used to easily add game states for testing purposes.
+ * FOR USE IN TESTING ONLY.
+ */
+export const testReplaceGameState = async (newGameState: GameState): Promise<GameState | undefined> => {
```

Comment:
can put this function under `dbTestHelpers.ts` or `dbHelper.ts` if it is only used for testing

Reply:
done, its in dbTestHelpers now.